### PR TITLE
[WM-2210] remove submission ux feature flag

### PR DESCRIPTION
**Pull request by kpierre13**
### What
- Removing the Submission UX feature flag and it's associations in files & tests

### Why
- All tickets to update the UX have been completed

### Testing strategy
- [x] Unit tests
- [x] Locally tested

 

[WM-2210]: https://broadworkbench.atlassian.net/browse/WM-2210?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/workflows-app/SubmissionHistory.js
Comment (dillydally414) [inline src/workflows-app/SubmissionHistory.js:1]: Should we also get rid of the route in this file? Right now users can still navigate to the submission history page.

File: src/workflows-app/SubmitWorkflow.js [hunk 1]
```diff
@@ -117,22 +85,19 @@ export const SubmitWorkflow = wrapWorkflowsPage({ name: 'SubmitWorkflow' })(
     const createWorkflowsApp = Utils.withBusyState(setCreating, async () => {
       try {
         setCreating(true);
-        if (isFeaturePreviewEnabled(ENABLE_AZURE_COLLABORATIVE_WORKFLOWS)) {
```
Comment (dillydally414) [inline src/workflows-app/SubmitWorkflow.js]: I think we want to leave this feature flag check in place

File: src/workflows-app/components/FindWorkflowModal.js
Comment (dillydally414) [inline src/workflows-app/components/FindWorkflowModal.js:1]: Nice

File: src/workflows-app/SubmissionHistory.js
Comment (kpierre13) [reply src/workflows-app/SubmissionHistory.js:1]: I think we should. The only thing I'm worried about is the breadcrumb from the `SubmissionDetails` page being broken. We can replace it temporarily to navigate to the landing page until a followup ticket fixes it?

File: src/workflows-app/SubmitWorkflow.js (hunk 1 above)
Comment (kpierre13) [reply src/workflows-app/SubmitWorkflow.js]: Oop, misread this flag. Thank you for catching!

**Review by dillydally414 — APPROVED** [review]

File: src/workflows-app/SubmissionDetails.js [hunk 2]
```diff
@@ -230,7 +230,7 @@ export const BaseSubmissionDetails = ({ name, namespace, workspace, submissionId
     const breadcrumbPathObjects = [
       {
         label: 'Submission History',
-        path: 'workspace-workflows-app-submission-history',
+        path: 'workspace-workflows-app', // TODO: Refactor to reroute to submission history tab in panel
```
Comment (mspector) [inline src/workflows-app/SubmissionDetails.js:233]: is there a ticket for this already? if so, might be useful to point to it in this comment!

**Review by mspector — APPROVED** [review]
👍

**Pull request merged**